src/recordings-ui.tsx

```diff
@@ -1,7 +1,7 @@
 import "./css/recordings.css";
 import DownloadImage from "./images/download.svg";
 import MediaPlayerImage from "./images/media_player.svg";
-import React, { Dispatch } from "react";
+import React /* , { Dispatch } */ from "react";
```

Comment:
`Dispatch` was declared but never used in this file. I figured there was some value in commenting it out rather than deleting it but I see how it clutters the code.

Reply:
Just as an aside here: as @mrose17 hints at, the general rule is to *never* commit *any* commented out code. It usually just serves to confuse the next person as when they encounter it and try to figure out _why_ it was commented out. If it's not needed, delete it. We use source control to track changes, so we can always look in the source control history to find code that has been deleted if needed. 

Happy with your update to this, so no further changes needed: just though it might be useful to outline this general guideline for future reference ;)